controller/sentry/service.go

```diff
@@ -0,0 +1,41 @@
+package sentry
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/giantswarm/microerror"
+)
+
+type Config struct {
+	Dsn string
+}
+
+type Service struct {
+	enabled bool
+}
+
+func New(config Config) (*Service, error) {
+	disabled := Service{enabled: false}
+	if config.Dsn == "" {
+		return &disabled, nil
+	}
+	err := sentry.Init(sentry.ClientOptions{
+		Dsn: config.Dsn,
+	})
+	if err != nil {
+		return &disabled, microerror.Mask(err)
```

Comment:
Best practise is to return the zero value in case an error is returned. Users must not expect to get anything in return in case an error is produced. 

```suggestion
		return nil, microerror.Mask(err)
```